cert/PowerShell_CACreationScript.ps1

```diff
@@ -0,0 +1,105 @@
+Invoke-Expression "openssl ca -create_serial -cert $root.crt -keyfile $root.pkey -outdir `"./root/ca/newcerts`" -batch -out $pi.crt -utf8 -days $intValidity -policy policy_anything -md sha384 -extensions $pi_sign -config `"$openssl\$cnf`" -in $pi.csr"
+
+#Professional Installer EE
+$pi_ee = "wif_pi_req"
+$pi_ee_sign = $pi_ee + $sig
+
+Invoke-Expression "openssl req -new -newkey rsa:$eeRSAKeysize -nodes -reqexts $pi_ee -out $pi_ee.csr -keyout $pi_ee.pkey -subj `"/C=US/ST=District of Columbia/L=Washington/O=Wireless Innovation Forum/OU=www.wirelessinnovation.org/CN=Installer End-Entity Example`" -config `"$openssl\$cnf`""
+Invoke-Expression "openssl ca -create_serial -cert $pi.crt -keyfile $pi.pkey -outdir `"./root/ca/newcerts`" -batch -out $pi_ee.crt -utf8 -days $eeValidity -policy policy_anything -md sha384 -extensions $pi_ee_sign -config `"$openssl\$cnf`" -in $pi_ee.csr"
+
+#PAL Intermediate
+$pal = "wif_pal_ca"
+$pal_sign = $pal + $sig
+
+Invoke-Expression "openssl req -new -newkey rsa:$intRSAKeysize -nodes -reqexts $pal -out $pal.csr -keyout $pal.pkey -subj `"/C=US/ST=District of Columbia/L=Washington/O=Wireless Innovation Forum/OU=www.wirelessinnovation.org/CN=WInnForum RSA PAL CA-1`" -config `"$openssl\$cnf`""
+Invoke-Expression "openssl ca -create_serial -cert $root.crt -keyfile $root.pkey -outdir `"./root/ca/newcerts`" -batch -out $pal.crt -utf8 -days $intValidity -policy policy_anything -md sha384 -extensions $pal_sign -config `"$openssl\$cnf`" -in $pal.csr"
+
+#PAL EE
+$pal_ee = "wif_pal_req"
+$pal_ee_sign = $pal_ee + $sig
+
+Invoke-Expression "openssl req -new -newkey rsa:$eeRSAKeysize -nodes -reqexts $pal_ee -out $pal_ee.csr -keyout $pal_ee.pkey -subj `"/C=US/ST=District of Columbia/L=Washington/O=Wireless Innovation Forum/OU=www.wirelessinnovation.org/CN=PAL End-Entity Example`" -config `"$openssl\$cnf`""
+Invoke-Expression "openssl ca -create_serial -cert $pal.crt -keyfile $pal.pkey -outdir `"./root/ca/newcerts`" -batch -out $pal_ee.crt -utf8 -days $eeValidity -policy policy_anything -md sha384 -extensions $pal_ee_sign -config `"$openssl\$cnf`" -in $pal_ee.csr"
+
+#Device Manufacturer Intermediate
+$dm = "wif_dm_ca"
+$dm_sign = $dm + $sig
+
+Invoke-Expression "openssl req -new -newkey rsa:$intRSAKeysize -nodes -reqexts $dm -out $dm.csr -keyout $dm.pkey -subj `"/C=US/ST=District of Columbia/L=Washington/O=Wireless Innovation Forum/OU=www.wirelessinnovation.org/CN=WInnForum RSA Device Manufacturer CA-1`" -config `"$openssl\$cnf`""
+Invoke-Expression "openssl ca -create_serial -cert $root.crt -keyfile $root.pkey -outdir `"./root/ca/newcerts`" -batch -out $dm.crt -utf8 -days $intValidity -policy policy_anything -md sha384 -extensions $dm_sign -config `"$openssl\$cnf`" -in $dm.csr"
+
+#Device Manufacturer EE
+$dm_ee = "wif_dm_req"
```

Comment:
Is this the intermediate for the device manufacturer or the CBSD (device) EE cert? If it's the EE cert, let's label it 'wif_cbsd_req'